addons/sourcemod/configs/szf/weapons.cfg

```diff
@@ -1,5 +1,365 @@
+		}
+		
+		"308" // Loch-n-Load
+		{
+			"rarity"	"uncommon"
+			"model"		"models/weapons/c_models/c_lochnload/c_lochnload.mdl"
+		}
+		
+		"996" // Loose Cannon
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_demo_cannon/c_demo_cannon.mdl"
+		}
+		
+		"1151" // Iron Bomber
+		{
+			"rarity"	"uncommon"
+			"model"		"models/workshop/weapons/c_models/c_quadball/c_quadball.mdl"
+		}
+		
+		"20" // Stickybomb Launcher
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_stickybomb_launcher/c_stickybomb_launcher.mdl"
+			"name"		"Stickybomb Launcher"
+		}
+		
+		"130" // Scottish Resistance
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_scottish_resistance/c_scottish_resistance.mdl"
+		}
+		
+		"1150" // Quickiebomb Launcher
+		{
+			"rarity"	"rare"
+			"model"		"models/workshop/weapons/c_models/c_kingmaker_sticky/c_kingmaker_sticky.mdl"
+		}
+		
+		"131" // Chargin Targe
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_targe/c_targe.mdl"
+		}
+		
+		"1099" // Tide Turner
+		{
+			"rarity"	"common"
+			"model"		"models/workshop/weapons/c_models/c_wheel_shield/c_wheel_shield.mdl"
+		}
+		
+		"141" // Frontier Justice
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_frontierjustice/c_frontierjustice.mdl"
+		}
+		
+		"527" // Widowmaker
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_dex_shotgun/c_dex_shotgun.mdl"
+		}
+		
+		"588" // Pomson
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_drg_pomson/c_drg_pomson.mdl"
+		}
+		
+		"997" // Rescue Ranger
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_tele_shotgun/c_tele_shotgun.mdl"
+		}
+		
+		"22" // Pistol
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_pistol/c_pistol.mdl"
+		}
+		
+		"140" // Wrangler
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_wrangler.mdl"
+		}
+		
+		"17" // Syringe Gun
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_syringegun/c_syringegun.mdl"
+		}
+		
+		"36" // Blutsauger
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_leechgun/c_leechgun.mdl"
+		}
+		
+		"305" // Crusader's Crossbow
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_crusaders_crossbow/c_crusaders_crossbow.mdl"
+		}
+		
+		"412" // Overdose
+		{
+			"rarity"	"common"
+			"model"		"models/weapons/c_models/c_proto_syringegun/c_proto_syringegun.mdl"
+		}
+		
+		"29" // Medigun
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_medigun/c_medigun.mdl"
+			"name"		"Medigun"
+		}
+		
+		"411" // Quick-Fix
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_proto_medigun/c_proto_medigun.mdl"
+		}
+		
+		"998" // Vaccinator
+		{
+			"rarity"	"rare"
+			"model"		"models/weapons/c_models/c_medigun_defense/c_medigun_defense.mdl"
+		}
+		
+		"14" // Sniper Rifle
+		{
+			"rarity"	"uncommon"
+			"model"		"models/weapons/w_models/w_sniperrifle.mdl"
```

Comment:
out of curiousity, is w_ used for backwards compatability? a c_ version exists under `models/weapons/c_models/c_sniperrifle/c_sniperrifle.mdl`.

Reply:
I didn't notice c_ version of sniper rifle exists, which i'd prefer that. Though we should also add w_ version under the "reskin" weapons list